packages/core/src/components/overlay/_overlay.scss

```diff
@@ -72,19 +72,14 @@ Styleguide components.overlay.scroll
 
```

Comment:
the words "absolute positioning" are used a few times in these docs, please update accordingly.